odpf/compass/v1beta1/service.proto

```diff
@@ -800,6 +800,16 @@ message SearchAssetsRequest {
     unique: true,
     ignore_empty: true
   }];
+
+  uint32 from = 7 [
```

Comment:
Can we rename this to `offset` to keep it consistent with `GetAllAssetsRequest1`?